.github/workflows/test.yml

```diff
@@ -16,12 +25,10 @@ jobs:
     strategy:
       matrix:
         cfg:
-          - { id: ubuntu-gcc, platform: ubuntu, cc: gcc, cpp: g++, setup_options: ""}
-          - { id: ubuntu-gcc-sanitize, platform: ubuntu, cc: gcc, cpp: g++, setup_options: "-Db_sanitize=address,undefined"}
-          - { id: ubuntu-clang, platform: ubuntu, cc: clang, cpp: clang++, setup_options: ""}
-          - { id: rockylinux-gcc, platform: ubuntu, cc: gcc, cpp: g++, setup_options: ""}
-          - { id: rockylinux-gcc-sanitize, platform: ubuntu, cc: gcc, cpp: g++, setup_options: "-Db_sanitize=address,undefined"}
-          - { id: rockylinux-clang, platform: ubuntu, cc: clang, cpp: clang++, setup_options: ""}
+          #- { id: ubuntu-gcc, platform: ubuntu, cc: gcc, cpp: g++, setup_options: "-Db_sanitize=address,undefined -Dunity=on -Dunity_size=12"}
+          - { id: ubuntu-clang, platform: ubuntu, cc: clang, cpp: clang++, setup_options: "-Dunity=on -Dunity_size=12"}
+          - { id: rockylinux-gcc, platform: rockylinux, cc: gcc, cpp: g++, setup_options: "-Dunity=on -Dunity_size=12"}
+          - { id: rockylinux-clang, platform: rockylinux, cc: clang, cpp: clang++, setup_options: "-Db_sanitize=address,undefined -Db_lundef=false -Dunity=on -Dunity_size=12"}
```

Comment:
I'm guessing one pass with the lowest common GCC and Clang versions (whatever OS that is) is probably fine if we wanted to fiddle around and save some compute. We'd probably miss some bugs here and there, but it would be occasional. Probably. I like merging this too though.

Reply:
This GCC in Ubuntu with sanitizers is hitting a bug somewhere, just not sure who's bug it is :)